src/content/structured/components/alerts/code.mdx

```diff
@@ -377,11 +377,135 @@ export const snippetsCustomMessage = [
         letterSpacing: "var(--ic-font-letter-spacing-0pt005)",
       }}
     >
-      Got to our{" "}
+      Go to our{" "}
       <IcLink href="#" onClick={(e) => e.preventDefault()}>
         coffee page
       </IcLink>{" "}
       to learn more.
     </span>
   </IcAlert>
 </ComponentPreview>
+
+### With custom icon
+
+export const snippetsCustomIcon = [
+  {
+    technology: "Web component",
+    snippets: {
+      short: `<ic-alert 
+  variant="neutral" 
+  heading="Increased wait times" 
+  show-default-icon="true"
```

Comment:
```suggestion
  show-default-icon="false"
```
should this be "false"? It doesn't seem right to set it to true but then supply a custom SVG to use